global/styles.js

```diff
@@ -13,6 +13,7 @@ export const colors = {
   white: '#FFFFFF',
   black: '#000000',
   almostBlack: '#111111', // prefer this for text color instead of black?
```

Comment:
Generally yes, but only when the contrast allows it. It would be cool to check it and refactor at some point...